src/components/List/index.tsx

```diff
@@ -1,48 +1,18 @@
 import React from "react";
 import "./index.scss";
 import Task from "../Task";
-import TaskType from "../../types/TaskType";
 import PriorityType from "../../types/PriorityType";
+import { useAppSelector } from "../../store/hooks";
 
-interface ListProps {
-  tasks: TaskType[];
-  setTasks: (tasks: TaskType[]) => void;
-  filter: PriorityType;
-}
-const List = ({ tasks, setTasks, filter }: ListProps) => {
-  const deleteTask = (id: string) => {
-    const newTasks = tasks.filter((task) => task.id !== id);
-    setTasks(newTasks);
-  };
-
-  const editTask = (updatedTask: TaskType) => {
-    const newTasks = tasks.map((task) => (task.id === updatedTask.id ? updatedTask : task));
-    setTasks(newTasks);
-  };
-
-  const completeTask = (id: string) => {
-    const newTasks = tasks.map((task) => {
-      if (task.id === id) {
-        return { ...task, completed: !task.completed };
-      }
-      return task;
-    });
-    setTasks(newTasks);
-  };
+const List = () => {
+  const filter = useAppSelector((state) => state.filter);
+  const tasks = useAppSelector((state) => state.tasksList.tasks);
```

Comment:
FYI you can add the below util to filterSlice file
`export const selectPriority = (state: RootState) => state.filter.priority;`
 and use it when you want like 
 `const priority = useAppSelector(selectPriority );`